django_postal_codes/models.py

```diff
@@ -187,65 +188,73 @@ class PostalCode(BaseModel):
         max_length=255,
         blank=True,
         null=True,
+        verbose_name=gettext_lazy("Artery type"),
     )
     prep1 = models.CharField(
         max_length=255,
         blank=True,
         null=True,
+        verbose_name=gettext_lazy("prep 1"),
     )
     artery_title = models.CharField(
         max_length=255,
         blank=True,
         null=True,
+        verbose_name=gettext_lazy("Artery title"),
     )
     prep2 = models.CharField(
         max_length=255,
         blank=True,
         null=True,
+        verbose_name=gettext_lazy("prep 2"),
     )
     artery_name = models.CharField(
         max_length=255,
         blank=True,
         null=True,
+        verbose_name=gettext_lazy("Artery name"),
     )
     artery_local = models.CharField(
         max_length=255,
         blank=True,
         null=True,
+        verbose_name=gettext_lazy("Artery local"),
     )
 
     postal_code = models.IntegerField(
-        verbose_name=_("Postal Code"),
+        verbose_name=gettext_lazy("Postal Code"),
         help_text="cp4",
         blank=True,
         null=False,
         db_index=True,
     )
     postal_code_extension = models.IntegerField(
-        verbose_name=_("Postal Code Extension"),
+        verbose_name=gettext_lazy("Postal Code Extension"),
         help_text="cp3",
         blank=True,
         null=False,
         db_index=True,
     )
     postal_designation = models.CharField(
         max_length=255,
-        verbose_name=_("Postal Designation"),
+        verbose_name=gettext_lazy("Postal Designation"),
         blank=True,
         null=True,
     )
 
     full_address = models.TextField(
-        verbose_name=_("Full address"),
+        verbose_name=gettext_lazy("Full address"),
         blank=True,
         null=False,
     )
 
-    def __str__(self) -> str:
+    def gettext_lazystrgettext_lazy(self) -> str:
         """
         String representation of this model.
         """
-        return _(f"{str(self.postal_code)}-{str(self.postal_code_extension).zfill(3)} / {self.full_address}")
+        return _(
```

Comment:
this wont translate. Can't use variables directly inside gettext
Need to use place holders and then % or format
Example:
Instead of 
```python
gettext(f"{self.my_field}")
```
It needs to be:
```python
gettext("{%s}") % (self.my_field,)
# or
 gettext("{my_field}").format(my_field=self.my_field)
```

Reply:
it is supposed to translate anything in this case? postal codes and address should not be translated.